recipes/default.rb

```diff
@@ -247,6 +247,14 @@
   action :nothing
 end
 
+local_systemd_dependencies = ""
+if service_discovery_enabled()
+  local_systemd_dependencies += "consul.service"
+end
+if exists_local("kafka", "default")
```

Comment:
okay so this will only work if online-fs runs on the same machine as the kafka broker?

Reply:
Yes, but it's not very important. That means start onlinefs after kafka but that doesn't guarantee that kafka is ready. So, it's OK to have it but not critical.